### PR TITLE
Ci pipeline

### PR DESCRIPTION
**Pull request by romain-grecourt**
First cut of a declarative pipeline for GlassFish.

### CODE REVIEW

File: nucleus/parent/pom.xml [hunk 1]
```diff
@@ -1120,4 +1121,17 @@
             <optional>true</optional>
         </dependency>
     </dependencies>
+    <repositories>
+        <repository>
```
Comment (vinayvishal) [inline nucleus/parent/pom.xml]: Do we need to have this maven.java.net repository configuration?

**Review by m0mus — APPROVED** [review]
LGTM

**Comment by m0mus** [comment]
@romain-grecourt Please sign your commit. It's a requirement for iP validation check.

**Review by m0mus — CHANGES_REQUESTED** [review]
Please sign your commit. It's a requirement for iP validation check.

**Comment by vinayvishal** [comment]
> Need to remake the PR to sign this off.
> Is the sign-off kept with squashing ?

One possibility could be, amend the last commit with sign off option and then force push your changes to the corresponding branch.

**Comment by romain-grecourt** [comment]
Thanks, sounds like a good option.
I'll force push to avoid re-creating a new PR.

**Pull request closed without merging**